mmpose/datasets/datasets/__init__.py

```diff
@@ -33,5 +33,5 @@
     'Face300WDataset', 'FaceAFLWDataset', 'FaceWFLWDataset', 'FaceCOFWDataset',
     'Body3DH36MDataset', 'AnimalHorse10Dataset', 'AnimalMacaqueDataset',
     'AnimalFlyDataset', 'AnimalLocustDataset', 'AnimalZebraDataset',
-    'AnimalATRWDataset'
+    'AnimalATRWDataset', AnimalPoseDataset
```

Comment:
What's the relationship between AnimalPoseDataset and AnimalZebraDataset?

Reply:
We have many dataset classes now, maybe its time to trim them down, and moving from code to configs.
Namely,  if user want to support a new dataset, they do not have to cp/paste/modify a dataset python file, instead they tweak a config file and voilà, works done.